Assets/MRTK/Core/Utilities/CameraFOVChecker.cs

```diff
@@ -37,7 +37,7 @@ public static bool IsInFOVCached(this Camera cam, Collider myCollider)
                 return false;
             }
 
-            Tuple<Collider, Camera> cameraColliderPair = new Tuple<Collider, Camera>(myCollider, cam);
+            ValueTuple<Collider, Camera> cameraColliderPair = ValueTuple.Create(myCollider, cam);
```

Comment:
ValueTuple is a .net framework 4.7 feature. any thoughts as to whether or not this will break on older unity versions?

Reply:
See: https://docs.microsoft.com/en-us/dotnet/api/system.valuetuple?view=net-6.0

Reply:
Validated a successful UWP IL2CPP build in Unity 2018 locally! 
![image](https://user-images.githubusercontent.com/3580640/173965056-866baf8a-6abd-40da-8adf-25385eda690d.png)